sycl/include/sycl/ext/intel/experimental/esimd/math.hpp

```diff
@@ -28,16 +28,46 @@ namespace intel {
 namespace experimental {
 namespace esimd {
 
+/// @addtogroup sycl_esimd_math
+/// @{
+/// @defgroup sycl_esimd_math_ext Hardware-accelerated math.
+///
+/// This is a group of APIs implementing standard math operations which are also
+/// directly supported by the hardware. Usually the hardware support is a
+/// specific message to the "extended math" GPU "shared function" unit, sent via
+/// the \c send or \c sends instructions. Most of the operations do not conform
```

Comment:
It's actually done with math instruction.

Reply:
right, fixed